packages/ProcurementML/src/ValueObjects/ProviderAiProvenance.php

```
@@ -0,0 +1,93 @@
+            throw ProcurementMlContractException::invalidValue('provider confidence');
+        }
+    }
+
+    /**
+     * @param array<string, mixed> $values
+     */
+    private function assertScalarMap(array $values, string $subject): void
+    {
+        foreach ($values as $key => $value) {
+            if (!is_string($key) || trim($key) === '') {
+                throw ProcurementMlContractException::invalidValue($subject);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\ProcurementML\Exceptions\ProcurementMlContractException' in method 'assertScalarMap'. (undefined)

(StaticAccess)